config/src/projects/habitat.ts

```diff
@@ -0,0 +1,79 @@
+import { Project } from './Project'
+import { tokenList } from '../tokens'
+
+export const habitat: Project = {
+  name: 'Habitat',
+  bridges: [
+    {
+      address: '0x96E471B5945373dE238963B4E032D3574be4d195',
+      sinceBlock: 12731872,
+      tokens: tokenList.map((e) => e.symbol),
+    },
+  ],
+  details: {
+    website: 'https://0xhabitat.org/',
+    color: '#29b65f',
+    technology: {
+      name: 'application specific optimistic-rollup',
+      details: 'NutBerry',
+    },
+    parameters: [
+      {
+        name: 'Primary use case',
+        value: 'Governance & Treasury Management',
+      },
+      {
+        name: 'Hypothetical level of decentralization',
+        value: 'High',
+        sentiment: 'good',
+      },
+      {
+        name: 'Current level of decentralization',
+        value: 'Low',
+        tooltip:
+          'Community owned Rollup, currently in the bootstrapping phase.',
+        sentiment: 'neutral',
+      },
+      {
+        name: 'Can funds be stolen by the operator?',
+        value: 'Yes, through a contract upgrade via the multisig.',
+        tooltip:
+          'Ownership will be transferred to community governance in the future.',
+        sentiment: 'neutral',
+        pointers: [
+          'https://github.com/0xHabitat/habitat/blob/master/src/rollup/contracts/HabitatV1.sol#L52',
+        ],
+      },
+      {
+        name: 'Permissionless?',
+        value: 'Yes',
+        sentiment: 'bad',
+      },
+      {
+        name: 'Force TX mechanism?',
+        value: 'Yes',
+        tooltip:
+          'Fully permissionless. Rollup node can be run inside web browsers.',
+        sentiment: 'good',
+      },
+      {
+        name: 'Privacy',
+        value: 'No',
+      },
+      {
+        name: 'Smart contracts',
+        value:
+          'Limited. Only static contracts that comply to the Habitat Module interface.',
```

Comment:
That is very interesting. I tried to find developer documentation for Habitat, but I was out of luck. Can you point me to some resources?

Reply:
We are currently working on that and only have more user-centric documentation @  https://docs.0xhabitat.org/

Reply:
Currently deployed modules serving as example: https://github.com/0xHabitat/habitat/blob/master/src/rollup/contracts/modules/

Reply:
Btw, this interface is going to include more `callbacks` in the future 🙂 . Right now it's only focused on governance proposals

Reply:
Interesting. So is my understanding correct here that the contracts are used as a configuration mechanism for voting? They need to be "pure" like pure functions, right? Also, can anyone deploy new ones?

Reply:
That's right. The actual validation / state storage is done by the rollup and are handled down to these contracts.

Reply:
> Also, can anyone deploy new ones
Yep. Anyone can register them. But this needs to happen on Layer 1 as data availability and verification source-of-truth. That's also why we do not need archive nodes for Ethereum mainnet. Ethereum light-nodes are sufficient for the NutBerry L2 node.